cli/cli.go

```diff
@@ -65,59 +64,148 @@ func (a *Application) init() {
 		},
 		Commands: []*cli.Command{
 			{
-				Name:  "add_peer",
-				Usage: "Invite peer or accept existing invitation from this peer",
+				Name:  "me",
+				Usage: "group of functions to work with your stats and settings",
+				Subcommands: []*cli.Command{
+					{
+						Name:   "stats",
+						Usage:  "print your stats",
+						Before: a.initApiConnection,
+						Action: func(c *cli.Context) error {
+							return printStatus(a.api)
+						},
+					},
+					{
+						Name:   "id",
+						Usage:  "print yours peer id",
+						Before: a.initApiConnection,
+						Action: func(c *cli.Context) error {
+							return printPeerId(a.api)
+						},
+					},
+				},
+			},
+			{
+				Name:  "peers",
+				Usage: "group of functions to work with peers. Use for check friend requests and peers status",
+				Subcommands: []*cli.Command{
+					{
+						Name:  "status",
```

Comment:
Result of this command doesn't fit into my screen :/
I have a few thoughts how to make it smaller
- we can fit Peer name/address/domain in one column, one value per line
- we can think about synonyms or something else for `disconnected`
- we can show total/rate network usage in one column